### PR TITLE
drop useless "!!! No reuse of etag" - fixes #13187

### LINKED ISSUE #13187: Remove bogus warning "no reuse of etag!!!"
When uploading/overwrite a file through WebDAV, the following warning/debug message often appears in the log:
```
!!! No reuse of etag for 'files/test.txt' !!!
```

From what I remember this message is harmless, we should look into removing it as it can be misleading.

@icewind1991 can you confirm ?

### PR DESCRIPTION
**Pull request by MorrisJobke**
cc @PVince81 @DeepDiver1975 @icewind1991

### CODE REVIEW

**Comment by MorrisJobke** [comment]
fixes #13187

**Comment by DeepDiver1975** [comment]
:+1:

**Comment by PVince81** [comment]
Part of the work is figuring out why we had this in the first place.

@icewind1991 can you comment ?

**Comment by DeepDiver1975** [comment]
> Part of the work is figuring out why we had this in the first place.

https://github.com/owncloud/core/issues/13187#issuecomment-69399772

**Comment by PVince81** [comment]
Fine by me :+1:

**Comment by PVince81** [comment]
@karlitschek backport to stable7 ?

File: lib/private/files/cache/scanner.php [hunk 1]
```diff
@@ -151,9 +151,6 @@ public function scanFile($file, $reuseExisting = 0) {
 					if (isset($newData['etag'])) {
```
Comment (DeepDiver1975) [inline lib/private/files/cache/scanner.php]: hmm - the whole if could have been removed ?

**Comment by DeepDiver1975** [comment]
> @karlitschek backport to stable7 ?

not that critical from my pov

**Comment by karlitschek** [comment]
not critical but also can´t break anything. Please backport

File: lib/private/files/cache/scanner.php (hunk 1 above)
Comment (MorrisJobke) [inline lib/private/files/cache/scanner.php]: I will prepare a PR against master and stable7

**Comment by PVince81** [comment]
stable7: 4d42485bf5a3d0af69d196a46c2b499112af515c

**Comment by MorrisJobke** [comment]
https://github.com/owncloud/core/pull/13316

**Pull request merged**